index.html

```diff
@@ -16,12 +17,27 @@
 
 <body>
 
- <!-- navigation code here -->
+ <!-- navigation code goes here -->
+
+ <header>
+  <div class="header-contents">
+    <h1>SpaceWalkers</h1>
+    <nav>
+      <a href="#">Home</a>
+      <a href="#">About Us</a>
+      <a href="#">Blog</a>
+      <a href="#">Contact</a>
+      <a href="#">Sign In</a>
+    </div>
+    </nav>
+
+  </header>
 
   <div class="container home">
 
     <header class="intro">
        <!-- image code here -->
+       <img class="banner"src="img/main-header.png" alt="Earthview from space">
```

Comment:
```suggestion
       <img class="banner"src="img/main-header.png" alt="Earthview from space" title="<enter something meaningful to a search engine here"/>
```
Think about adding the title property to your img tags to improve SEO.